modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -255,7 +255,7 @@ internal static void SetGodotObjectPtr(IntPtr gcHandlePtr, IntPtr newPtr)
                     .FirstOrDefault(a => a.GetName().Name == "GodotSharpEditor");
                 wrapperType = editorAssembly?.GetType("Godot." + nativeTypeNameStr);
 
-                if (wrapperType == null)
+                if (editorAssembly != null && wrapperType == null)
```

Comment:
What do you think about splitting the `if` and merging with the previous `wrapperType` assignment like this?

```csharp
if (editorAssembly != null)
{
	wrapperType = editorAssembly.GetType("Godot." + nativeTypeNameStr);

	if (wrapperType == null)
	{
		wrapperType = GetTypeByGodotClassAttr(editorAssembly, nativeTypeNameStr);
	}
}
```

Reply:
Splitting makes more sense, seeing as it can't even do the wrapper type step if null